pillar/cni.sls

```diff
@@ -2,6 +2,7 @@
 flannel:
   image:          'sles12/flannel:0.9.1'
   backend:        'vxlan'   # UDP seems to be near end of life (https://github.com/coreos/flannel/pull/786)
```

Comment:
Oh, also this comment can be removed, as we aren't using the `udp` backend anymore, just realized about this comment.